MonkeyBot/Database/MonkeyDBContext.cs

```diff
@@ -12,7 +13,7 @@ public class MonkeyDBContext : DbContext
         public DbSet<BenzenFact> BenzenFacts { get; set; }
         public DbSet<GuildConfig> GuildConfigs { get; set; }
         public DbSet<TriviaScore> TriviaScores { get; set; }
-        public DbSet<Announcement> Announcements { get; set; }
+        public DbSet<Reminder> Reminder { get; set; }
```

Comment:
Can we please pluralize this, so it is consistent with the others? (`Reminder` -> `Reminders`)